bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaActionGroupHandler.java

```diff
@@ -34,8 +34,16 @@ public SomfyTahomaActionGroupHandler(Thing thing) {
     }
 
     @Override
-    protected boolean isAlwaysOnline() {
-        return true;
+    public void initializeThing(@Nullable ThingStatus bridgeStatus) {
+        if (bridgeStatus != null) {
+            if (bridgeStatus == ThingStatus.ONLINE) {
+                updateStatus(ThingStatus.ONLINE);
+            } else {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+            }
+        } else {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED);
+        }
```

Comment:
If this Thing should be online if and only if the bridge is online. You can simply set it to ONLINE in `initialize()`. The framework takes care of the state, then.

Reply:
This method overrides a more complex method used for all things defined in the parent class `SomfyTahomaBaseThingHandler`.

Reply:
I see. `bridgeStatusChanged()` is overriden.

There's a HTTP request done in `initialize()`. Would be awesome if you could fix this in another PR!

Reply:
Yes, I will do it in another PR.

Reply:
Guys, so what's wrong with this code, should I change something in this PR?

Reply:
Nothing to change un m'y opinion.